dart/tool/java/generate_java.dart

```diff
@@ -62,11 +84,16 @@ class Api extends Member with ApiParseUtil {
     _setFileHeader();
 
     // Add undocumented "id" property
+    addProperty('InstanceRef', 'id', javadoc: 'The id of this instance.');
+    addProperty('Instance', 'id', javadoc: 'The id of this instance.');
     addProperty('LibraryRef', 'id', javadoc: 'The id of this library.');
+    addProperty('Library', 'id', javadoc: 'The id of this library.');
     addProperty('ScriptRef', 'id', javadoc: 'The id of this script.');
+    addProperty('Script', 'id', javadoc: 'The id of this script.');
```

Comment:
`Script` extends `Obj`, which is where it gets it's `id` field. https://github.com/dart-lang/vm_service_drivers/blob/master/dart/lib/vm_service_lib.dart#L1589

The `superName` field is being preserved - perhaps the java generator isn't using it?